healthcare/api-client/fhir/fhir_resources_test.py

```diff
@@ -28,6 +28,7 @@
 project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
 service_account_json = os.environ["GOOGLE_APPLICATION_CREDENTIALS"]
 
+bundle = os.path.join(os.path.dirname(__file__), "resources/execute_bundle.json")
 dataset_id = "test_dataset_{}".format(int(time.time()))
```

Comment:
I'd recommend using the UUID module instead - I think it's possible that the resources run by the tests in parallel are created at the same time and are colliding and that's what's causing various 404s

Reply:
Good point, thanks. Done.